framework_tvb/tvb/interfaces/web/static/js/projectTree.js

```diff
@@ -4,7 +4,7 @@
  * TheVirtualBrain-Scientific Package (for simulators). See content of the
```

Comment:
I think that most of the JS files have a wrong header. 
It currently says "TheVierualBrain-Scientific package..." but it should be the text with "TheVirtualBrain-Framework Package. ....." (like in users_controller.py for example).
Could you please fix them?
All files under "framework_tvb" should have the header with the "...Framewrok Package ..."

Reply:
In some places the whole first paragraph is wrong, for example in fcd_adapter. I think only the first paragraph is different for the library and framework tests.

Reply:
Correct, only the first paragraph is different between library and framework.

For tvb-build, tvb-documentation, tvb-rest, etc the header can be the same as the one in framework.

Reply:
You are right about

> I didn't see JS files with the wrong header, they actually started with "TheVirtualBrain-Framework Package.".. I completed the headers where the CITATION part was missing.

You are right about the header, is not switched in JS and CSS files. I didn't look carefull enough before.
I think that in the CSS and JS files, the citation was deliberately left out, to minimize the size. If it is not too much trouble I would in fact prefer to keep it out ...  :-|

Reply:
Ok, but there were some js and css files were the citation was left there.  I am going to remove the citation from each js and css files then.